installer_win/omsupply_web_server.suf

```diff
@@ -0,0 +1,4455 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
:nit the name of the suf file isn't quite accurate: the remote server is more than a web server, so the suf files would be better named just `omsupply_server.suf` and `omsupply_server_upgrader.suf`

Reply:
I was trying to emphasize that the installer is for web interface based server so used "web" word in it - though it does sounds like or confuses that it is just only a normal web server.
5c5269893